src/devices/depthCamera/depthCameraDriver.cpp

```diff
@@ -744,22 +537,22 @@ bool depthCameraDriver::setRgbFOV(double horizontalFov, double verticalFov)
 
 bool depthCameraDriver::setDepthFOV(double horizontalFov, double verticalFov)
 {
-    if (m_cameraDescription->depth_Fov.isDescription)
+    if (m_params_map[depth_Fov].isDescription)
     {
         return false;
     }
     return setFOV(horizontalFov, verticalFov, m_depthStream);
 }
 
-bool depthCameraDriver::setDepthAccuracy(double accuracy)
+bool depthCameraDriver::setDepthAccuracy(double _accuracy)
 {
-    if (m_cameraDescription->accuracy.isDescription)
+    if (m_params_map[accuracy].isDescription)
```

Comment:
Didn't go through the code and from the PR it is not immediate to check this out but is `accuracy` chosen on purpose in place of `_accuracy`?

Reply:
Yes because this function is called also in the configuration phase, and in case the accuracy is present as description in the ini file the set has to be skipped :wink:

Reply:
Then you might want to highlight this fact by using `this->accuracy`.